VHbbAnalysis/Heppy/test/combined_cmssw.py

```diff
@@ -647,11 +647,26 @@ def initialize(**kwargs):
         setupAllVIDIdsInModule(process, eleid, setupVIDElectronSelection)
     process.OUT.outputCommands.append("keep *_electronMVAValueMapProducer_*_EX")
     process.OUT.outputCommands.append("keep *_egmGsfElectronIDs_*_EX")
+
+    
+
+    ########################################
+    # MET significance matrix
+    ########################################
     
+    from PhysicsTools.PatUtils.tools.runMETCorrectionsAndUncertainties import runMetCorAndUncFromMiniAOD
+    isData = not isMC
+    runMetCorAndUncFromMiniAOD(process, isData = isData)
+    process.OUT.outputCommands.append("keep *_slimmedMETs_*_EX")
+
+    processDumpFile = open('combined_cmssw.dump', 'w')
```

Comment:
can you comment on the fact that MET is changing?

On Thu, Sep 1, 2016 at 9:19 AM, Christian Veelken <notifications@github.com>
wrote:

> In VHbbAnalysis/Heppy/test/combined_cmssw.py
> <https://github.com/vhbb/cmssw/pull/524#discussion_r77125006>:
>
> >
> > +    from PhysicsTools.PatUtils.tools.runMETCorrectionsAndUncertainties import runMetCorAndUncFromMiniAOD
> > +    isData = not isMC
> > +    runMetCorAndUncFromMiniAOD(process, isData = isData)
> > +    process.OUT.outputCommands.append("keep *_slimmedMETs_*_EX")
> > +
> > +    processDumpFile = open('combined_cmssw.dump', 'w')
>
> Hi Andrea, I have removed the python dump from PR #527
> <https://github.com/vhbb/cmssw/pull/527>. Cheers, Christian
> … <#m_-4376965942623735700_>
> On Aug 31, 2016, at 6:32 PM, arizzi wrote: In VHbbAnalysis/Heppy/test/combined_cmssw.py:
> > > + from PhysicsTools.PatUtils.tools.runMETCorrectionsAndUncertainties
> import runMetCorAndUncFromMiniAOD > + isData = not isMC > +
> runMetCorAndUncFromMiniAOD(process, isData = isData) > +
> process.OUT.outputCommands.append("keep *_slimmedMETs_*_EX") > + > +
> processDumpFile = open('combined_cmssw.dump', 'w') no, this is creating a
> dump file and printing it... why? � You are receiving this because you
> authored the thread. Reply to this email directly, view it on GitHub, or
> mute the thread.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/vhbb/cmssw/pull/524/files/3cee527c2453708eaaf614f591ef95c72e08d610#r77125006>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AEyilp9UXYY9lImc7afVmWQC6ODkEyHoks5qlnydgaJpZM4JruKR>
> .
>